R/deps.R

```diff
@@ -407,7 +429,13 @@ standardise_dep <- function(x) {
   } else if (identical(x, FALSE)) {
     character(0)
   } else if (is.character(x)) {
-    x
+    if (any(x %in% c("hard", "user"))) {
+      c("Depends", "Imports", "LinkingTo")
+    } else if (any(x %in% c("soft", "dev"))) {
+      c("Depends", "Imports", "LinkingTo", "Suggests")
```

Comment:
`Enhances`?

Reply:
I was treating this like `dependencies = TRUE`, which does _not_ seem to include `Enhances`